common/artifacts/904_wc_artifacts_amulets_generic.txt

```diff
@@ -8,11 +8,11 @@ amulet_5_diplomacy = {
 
 	diplomacy = 3
 
-	general_opinion = 4
-	vassal_opinion = 4
-	same_religion_opinion = 8
+	general_opinion = 5
+	vassal_opinion = 5
+	same_religion_opinion = 15
 
-	monthly_character_prestige = 1
+	monthly_character_prestige = 0.5
```

Comment:
+25 opinion of vassals of your religion. Kinda too much... About +15 is fine.
Prestige used to be a significant part of the cost.

Reply:
changed